src/main/scala/io/citrine/lolo/bags/BaggedResult.scala

```diff
@@ -403,9 +403,7 @@ case class MultiPredictionBaggedResult(
 case class MultiTaskBaggedResult(
                                   baggedPredictions: Seq[BaggedResult[Any]],
                                   realLabels: Seq[Boolean],
-                                  NibIn: Vector[Vector[Int]],
-                                  trainingLabels: Seq[Seq[Any]],
-                                  trainingWeights: Seq[Double]
```

Comment:
These two arguments were being piped through to calculate the correlation coefficient using the training data. This is no longer necessary.